api/database/auth.ts

```diff
@@ -4,8 +4,10 @@ const { getFirestore, Timestamp, FieldValue } = require('firebase-admin/firestor
 var serviceAccount: object;
 
 try {
-  serviceAccount = require('../../serviceAccountKey.json');
+  serviceAccount = require('../../api/database/serviceAccountKey.json');
```

Comment:
Since this is a runtime file dependency it should not be hard-coded. I recommend adding a .env entry for the location of the serviceAccountKey